Assembly/Whiskey/Tasks/ValidatePathAttribute.cs

```diff
@@ -7,5 +7,11 @@ public sealed class ValidatePathAttribute : Attribute
         public bool Mandatory { get; set; }
 
         public string PathType { get; set; }
+
+        public bool MustExist { get; set; } = true;
+        
+        public bool AllowAbsolute { get; set; }

```

Comment:
We should always allow absolute paths as long as they point to a path inside the build root. So, remove this property.